Source/SwiftLintFramework/Rules/PrefixedTopLevelConstantRule.swift

```diff
@@ -54,6 +54,11 @@ public struct PrefixedTopLevelConstantRule: ASTRule, OptInRule, ConfigurationPro
     public func validate(file: File,
                          kind: SwiftDeclarationKind,
                          dictionary: [String: SourceKitRepresentable]) -> [StyleViolation] {
+        if self.configuration.onlyPrivateMembers,
+            let acl = dictionary.accessibility.flatMap(AccessControlLevel.init(identifier:)), !acl.isPrivate {
+			return []
```

Comment:
The indentation here looks a little bit odd.

Reply:
huh yea, tab got in here some how